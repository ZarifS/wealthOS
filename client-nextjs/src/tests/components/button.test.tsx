import { render } from '@testing-library/react';
import Button from '../../components/button';

describe('Button component', () => {
    it('renders correctly', () => {
        const { asFragment } = render(
            <Button text="Click me" onClick={() => { }} />
        );

        expect(asFragment()).toMatchSnapshot();
    });
});
