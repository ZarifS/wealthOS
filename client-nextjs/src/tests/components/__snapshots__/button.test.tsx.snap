// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component renders correctly 1`] = `
<DocumentFragment>
  <button
    class="customButton "
    type="button"
  >
    Click me
  </button>
</DocumentFragment>
`;
